src/toolbar/sticky/stickytoolbarview.js

```diff
@@ -44,15 +108,6 @@ export default class StickyToolbarView extends ToolbarView {
 	init() {
 		super.init();
 
-		/**
-		 * A dummy element which visually fills the space as long as the
-		 * actual toolbar is sticky. It prevents flickering of the UI.
-		 *
-		 * @private
-		 * @property {HTMLElement} ui.stickyToobar.StickyToolbarView#_elementPlaceholder
-		 */
-		this._elementPlaceholder = document.createElement( 'div' );
-		this._elementPlaceholder.classList.add( 'ck-toolbar__placeholder' );
 		this.element.parentNode.insertBefore( this._elementPlaceholder, this.element );
 
 		// Update sticky state of the toolbar as the window is being scrolled.
```

Comment:
Looks like there is no need to listen on `scroll` event all the time. You can depend it on `isActive` state:
```javascript
this.listenTo( this.model, 'change:isActive', ( evt, name, value ) => {
  if ( value ) {
    this.listenTo( window, 'scroll', () => this._checkIfShouldBeSticky() );
  } else {
    this.stopListening( window );
  }

  this._checkIfShouldBeSticky();
} );
```